spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
```

Comment:
- [ ] 10 Tests are failing, kindly update them
![image](https://user-images.githubusercontent.com/62705947/193235512-da586282-ae45-4247-9a0d-65436e6d1d08.png)

Reply:
please test for integration as in the pr message

Reply:
All tests work in the last branch(f/forms)

Reply:
Integration test software is affecting model and requests tests

Reply:
I'm stuck here, please help out

Reply:
Okay, thanks. I am requesting another review